.env

```diff
@@ -13,4 +13,4 @@ NEXT_PUBLIC_SENTRY_DSN=https://3f5bb9023ff0407299dd22a6454558f9@o450487248892723
 # For running the API local, use your devices localhost address:
 #
 # NEXT_PUBLIC_API_URL=http://127.0.0.1:3001/v2
-# NEXT_PUBLIC_EXP_API_URL=http://127.0.0.1:3001/v2
+# NEXT_PUBLIC_EXP_API_URL=http://127.0.0.1:3001/v2
```

Comment:
refactor (blocking): This change shouldn't be part of the PR

Reply:
Removed in latest commit; thanks for spotting that!

Reply:
Did you mean that line or the entire file? I see `.env` in the `.gitignore` file and am surprised it'd include this in the PR.